docs/manual/format_v3.md

```diff
@@ -56,6 +56,8 @@ contains an OpenPGP signature on the header + payload data. The PGP
 tag is used for RSA signatures and the GPG tag is used for DSA
 signatures.
 
+Note: the signature tags overlap with those of the main header.
```

Comment:
The precise meaning of this is a bit ambiguous.  It might be possible to read this as "some tags can exist in both headers simultaneously" or "some tags can exist in either header".  The latter is technically true but isn't that more an artifact of how the headers get merged together than "correctness"?

It's definitely not universally true, as something would have to be seriously broken for a main header checksum/signature to end up in the main header :)

Reply:
Oh, and as per https://github.com/rpm-software-management/rpm/issues/2462#issuecomment-1493217258, `RPMTAG_FILESIGNATURES` and `RPMSIGTAG_FILESIGNATURES` are actually completely different non-overlapping constants, it's just not noticed because of, again, the merging of the signature and main headers during queries.

Reply:
What I mean by the overlap is that the tag values clash, eg 1000 in a signature means size, but name in the main header. In v3 (which is what we're commenting right here), ALL the signature tags clash with the main tags this way. Most of the newer v4 tags don't, but unfortunately RPMSIGTAG_RESERVEDSPACE was added in the main header range.